hive/tests/conftest.py

```diff
@@ -19,4 +19,7 @@ def dd_environment():
         log_patterns=[r'datanode:\(\d+\) is available', 'Starting Hive Metastore Server', 'Starting HiveServer2'],
         sleep=2,
     ):
-        yield load_jmx_config(), {'use_jmx': True}
+        instance = {'host': 'localhost', 'port': 8809}
```

Comment:
Let's be explicit about the instances we are using instead of using the config from conf.example.yaml.